### PR TITLE
Services validation

### PR DESCRIPTION
**Pull request by Cifixie**
Just basic functionality - MWP for tomorrow. Nothing fancy...

### CODE REVIEW

**Comment by artur-gajewski** [comment]
Some styling issues:

screenshot 2018-12-11 at 7 53 41

screenshot 2018-12-11 at 8 20 01

File: src/components/berths/SelectedBerth.js [hunk 1]
```diff
@@ -40,39 +41,105 @@ const DeselectButton = styled.button`
...
-            <Col xs={12} sm={6}>
-              <StyledButton
-                type="button"
-                onClick={() => moveDown(berth.identifier)}
-                disabled={last}
+const Invalid = styled.div`
+  position: relative;
+  display: inline-block;
+`;
+const ToolTip = styled.div`
```
Comment (artur-gajewski) [inline src/components/berths/SelectedBerth.js]:
```suggestion

const ToolTip = styled.div`
```

File: src/components/berths/SelectedBerth.js [hunk 2]
```diff
@@ -40,39 +41,105 @@ const DeselectButton = styled.button`
...
+class SelectedBerth extends Component {
+  state = {
+    show: false
+  };
+
+  toggle = visibility => this.setState(() => ({ show: visibility }));
+
+  render() {
+    const { show } = this.state;
+    const {
```
Comment (artur-gajewski) [inline src/components/berths/SelectedBerth.js]:
```suggestion

    const {
```

File: src/components/berths/SelectedBerth.js [hunk 3]
```diff
@@ -40,39 +41,105 @@ const DeselectButton = styled.button`
...
-        </Container>
-      </BerthOptions>
-    </StyledRow>
-  </Container>
-);
+                <Icon color="red" name="commenting" width="1em" height="1em" />
+                <ToolTip show={show}>
+                  {notSuitable && <span>DOES NOT MATCH WITH BOAT TYPE</span>}
+                  <ul>
+                    {missingServices.map(a => (
```
Comment (artur-gajewski) [inline src/components/berths/SelectedBerth.js]:
```suggestion
                    {missingServices.map(service => (
```

File: src/components/berths/SelectedBerth.js [hunk 4]
```diff
@@ -40,39 +41,105 @@ const DeselectButton = styled.button`
...
+                      <li key={a}>{a}</li>
```
Comment (artur-gajewski) [inline src/components/berths/SelectedBerth.js]:
```suggestion
                      <li key={service}>{service}</li>
```

File: src/components/berths/SelectedBerth.js [hunk 5]
```diff
@@ -40,39 +41,105 @@ const DeselectButton = styled.button`
...
-            </Col>
-          </Row>
-        </Container>
-      </BerthOptions>
-    </StyledRow>
-  </Container>
-);
+                <Icon color="red" name="commenting" width="1em" height="1em" />
+                <ToolTip show={show}>
+                  {notSuitable && <span>DOES NOT MATCH WITH BOAT TYPE</span>}
```
Comment (artur-gajewski) [inline src/components/berths/SelectedBerth.js]: Intl

File: src/components/berths/SelectedBerth.js (hunk 4 above)
Comment (artur-gajewski) [reply src/components/berths/SelectedBerth.js]: Also, localize rendered service

**Comment by Cifixie** [comment]
Not going to fix the UI until we have decided what to do with @terotic so going to leave them as they are now. If not merged, going to implement the design with in this branch

**Pull request merged**